hello-parsec.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env sh
+
+# Copyright 2022 Contributors to the Parsec project.
+# SPDX-License-Identifier: Apache-2.0
+
+ping_parsec() {
+    echo "Checking for availability of the Parsec service on your system... "
+    $PARSEC_TOOL ping > /dev/null
+}
+
+PARSEC_SERVICE_ENDPOINT="${PARSEC_SERVICE_ENDPOINT:-unix:/run/parsec/parsec.sock}"
+PARSEC_TOOL="${PARSEC_TOOL:-$(which parsec-tool)}"
+KEY_NAME=HelloParsecDemoKey
+export RUST_LOG="${RUST_LOG:-error}"
+
+if ! ping_parsec; then exit 1; fi
+
+cat ./parsec_banner.txt
+
+echo "Parsec back-end providers enabled (the topmost entry is the default):-"
+$PARSEC_TOOL list-providers 2>/dev/null | grep "^ID:" | grep -v "0x00"
+
+echo "Running RSA encryption demo. Three 'Hello Parsec' messages should appear below..."
+echo
+
+$PARSEC_TOOL create-rsa-key --key-name $KEY_NAME
```

Comment:
Just a suggestion, would it be useful to show these commands, so that the developer could have a direct feel of what parsec-tool commands are executed, e.g.:
```
echo "Executing: $PARSEC_TOOL create-rsa-key --key-name $KEY_NAME"
$PARSEC_TOOL create-rsa-key --key-name $KEY_NAME

echo "Executing: $PARSEC_TOOL encrypt --key-name $KEY_NAME \"Hello Parsec from Rust!\""
$PARSEC_TOOL encrypt --key-name $KEY_NAME "Hello Parsec from Rust!" | $HELLO_DECRYPT_RUST

...
```

Reply:
Later I will add parsec-cli-test.sh script to this container. This script runs E2E Parsec tests and there will be possibility to see all the commands run.
By default we would like to keep the output as simple as possible

Reply:
Actually the Dockerfile already pulls the CLI tests into the container, but does not provide a means to execute them as yet.